test/TesterInternal/GrainReferenceCastTests.cs

```diff
@@ -19,8 +19,8 @@ public void CastGrainRefCastFromMyType()
         {
             GrainReference grain = (GrainReference)GrainClient.GrainFactory.GetGrain<ISimpleGrain>(random.Next(), SimpleGrain.SimpleGrainNamePrefix);
             GrainReference cast = (GrainReference)grain.AsReference<ISimpleGrain>();
-            Assert.IsInstanceOfType(cast, grain.GetType());
-            Assert.IsInstanceOfType(cast, typeof(ISimpleGrain));
+            Assert.IsType(grain.GetType(), cast);
+            Assert.IsType<ISimpleGrain>(cast);
```

Comment:
I believe that this test class was not correctly migrated. `Assert.IsType` is not equivalent to MsTest's `Assert.IsInstanceOfType`. 
`Assert.IsType` checks that the type is exactly that passed in. Here (and in tests below) we are asserting that the reference implements the different interfaces, so it will certainly fail. You probably want to use `Xunit.Assert.IsAssignableFrom`, which is equivalent to the one from MsTest